### ISSUE TITLE
What is *** setObjectForKey: key cannot be nil

### ISSUE THREAD

**Issue by CristianCardosoA** [BlueM/cliclick#47]
I cannot execute any command.

**Comment by BlueM** [comment]
That’s a bug. Which version were you using? The one from www.bluem.net, from Homebrew, from MacPorts or did you clone the repository and built your own?

And what command exactly did you execute?

**Comment by CristianCardosoA** [comment]
I cloned the repository - master branch from GitHub.

I'm trying to run a simple command like in the documentation : cliclick c:12,34.

**Comment by BlueM** [comment]
What OS version?
How did you build? Xcode or `make`?
Can you confirm it works flawlessly when compiling using Xcode, but shows the problem you describe when using `make`?

**Comment by CristianCardosoA** [comment]
I'm using Xcode 8.2. 
At the very first time that I tried to run "`xcodebuild`" I got this error: 

> === BUILD TARGET cliclick OF PROJECT cliclick WITH THE DEFAULT CONFIGURATION (Release) ===
> 
> Check dependencies
> error: There is no SDK with the name or path '/Users/developer/Downloads/cliclick-master/macosx10.7'
> 
Then I opened Xcode and updated the recommend settings and then perform changes. After that I finally made a build successfully. And finally I executed both commands, **`sudo make install`** and **`xcodebuild`**. 

I try also `make` command, and I didn't get any error. When I executed it again and I get this `./generate-action-classes-macro.sh
`
